Sources/CustomerCenter/CustomerCenterConfigData.swift

```diff
@@ -335,19 +335,22 @@ public struct CustomerCenterConfigData: Equatable {
         public let openMethod: OpenMethod?
         public let type: PathType
         public let detail: PathDetail?
+        public let refundWindow: RefundWindowDuration?
```

Comment:
Hmm I'm not sure I like `refundWindow` as the name, it's not clear it's a duration from the name. Should we name it the full type name: `refundWindowDuration`?

TBH, it's unlikely this is going to be used by anyone other than us, but still I think it would be nicer to make it clearer... Wdyt?